lib/pages/login/login_page.dart

```diff
@@ -102,9 +101,7 @@ class LoginPage extends StatelessWidget {
             ),
           ],
         ),
-        SizedBox(
-          height: 20.0,
-        ),
+        SizedBox(height: Dimens.defaultMarginPadding),
```

Comment:
consider `const SizedBox` so it will not be rebuilt ever, because it's a static constant space anyway:

read more: https://dev.to/pedromassango/flutter-performance-tips-1-const-constructors-4j41

Reply:
Fixed https://github.com/luongvo/flutter-survey/pull/49/commits/199222203d41fcda6b92cd4385ab6b06d04636f1